<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
    <item row="4" column="0">
     <widget class="QPushButton" name="mapTest">
      <property name="text">
       <string>TestMap</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLineEdit" name="startcity">
      <property name="placeholderText">
       <string>Enter Start City</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QPushButton" name="submit">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_teste_was">
      <property name="placeholderText">
       <string>Ihre Eingabe</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="dijkstraevaluation">
      <property name="text">
       <string>Dijkstra Evaluation</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_teste_was">
      <property name="toolTip">
       <string>Testet</string>
      </property>
      <property name="text">
       <string>Teste was</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QComboBox" name="stadtList">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Koeln</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bonn</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Duesseldorf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Aachen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Clear All</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="testAbstractMap">
      <property name="text">
       <string>Test Abstract Map</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="fillMap">
      <property name="text">
       <string>FillMap</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="testDrawStreets">
      <property name="text">
       <string>TestStreets</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLineEdit" name="endcity">
      <property name="placeholderText">
       <string>Enter Final Destination</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="19">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="drawcitytest">
      <property name="text">
       <string>Test Draw City</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="addStreet">
      <property name="text">
       <string>Add Street</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="addCity">
      <property name="text">
       <string>AddCity</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_eingabe">
      <property name="text">
       <string>Ihre Eingabe</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Find the dijkstra:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuScene">
     <property name="title">
      <string>Scene</string>
     </property>
     <addaction name="actionClear_Scene"/>
    </widget>
    <widget class="QMenu" name="menuInfo">
     <property name="title">
      <string>Info</string>
     </property>
     <addaction name="actionAbout"/>
    </widget>
    <addaction name="actionFile"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExid"/>
    <addaction name="menuScene"/>
    <addaction name="menuInfo"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClear_Scene">
   <property name="text">
    <string>Clear Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExid">
   <property name="text">
    <string>Exid</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
